pkg/client/orm/qb_postgres.go

```diff
@@ -0,0 +1,218 @@
+package orm
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var Quote string = `"`
```

Comment:
private variable or not? if private , change scope pls

Reply:
ok